app/src/main/java/com/zeapo/pwdstore/crypto/PasswordCreationActivity.kt

```diff
@@ -258,7 +258,7 @@ class PasswordCreationActivity : BasePgpActivity(), OpenPgpServiceConnection.OnB
         }
         if (maybeLongKeyId != null) {
             val keyId = maybeLongKeyId.toULong()
```

Comment:
And this is where the code currently explodes, being unable to parse our hexadecimal key ID. The key ID in question is `b7843f823355e9b9`.

Reply:
This should be fixed by https://github.com/android-password-store/Android-Password-Store/pull/959, which we could merge first.